### DISCUSSION TITLE
Great job on Embedding

### DISCUSSION THREAD

**Discussion by hittjwX** [brianpetro/obsidian-smart-connections#722] [General]
If nobody has said it, great job on the more recent embedding. I keep the `Smart Connections File` window open when working sometimes. It used to kick off rebuilds as documented changes or switching between documents.

This would have caused delays. Now it goes very fast. The *date modified* in the `.smart-connections` folder only embeds what has changed. It's more real-time without the build delays.

If I close the side bar, I get a little speed boost. The embed dialog doesn't come up -- which is awesome because I need to the horsepower for writing. Thank you for this efficiency. In the past embedding would distract from the note-taking process.

**Comment by brianpetro** [comment]
Thanks for noticing 😊🌴
